### ISSUE TITLE
Can you add a option to summaryze youtube videos from Transcript?

### ISSUE THREAD

**Issue by falberto** [taishi55/chatgpt-phantom#8]

**Comment by taishi55** [comment]
You can add a custom instruction to summarize it. For example, summarize the given info.

**Comment by falberto** [comment]
It would be cool to pass a link of a video, the tool performs the download of the transcript and then performs the summary, do you think it's possible?

**Comment by taishi55** [comment]
If you copy & paste the video title and the account name to filter even more, I think you can still get the expected result. 

Yes that's an easy modification for me. I will add the function for the next update.

**Comment by Loflou** [comment]
Hi, I love the extension btw! Would it be possible to just start off with the URL search box from the beginning? Maybe with a command or something? Also can you make the URL search any link instead of just youtube? Thanks!

**Comment by taishi55** [comment]
@Loflou Yes adding a command is possible, yet this extension is designed for non-tech users. I'm not sure if it's useful for many users. 

> can you make the URL search any link instead of just youtube?
Yes, I will add the feature!
